Remove debug code from tests (since they'd require USE=debug)

http://bugs.gentoo.org/286173
http://bugs.gentoo.org/288156

diff -r f5754f5d165c -r 3aa1d93d1c10 tests/Test-idn-cmd.px
--- a/tests/Test-idn-cmd.px	Fri Sep 25 10:17:49 2009 -0700
+++ b/tests/Test-idn-cmd.px	Sun Sep 27 11:29:01 2009 -0700
@@ -28,7 +28,7 @@
     },
 );
 
-my $cmdline = $WgetTest::WGETPATH . " --debug --iri -rH"
+my $cmdline = $WgetTest::WGETPATH . " --iri -rH"
     . " -e http_proxy=localhost:{{port}} --local-encoding=EUC-JP $euc_jp_hostname";
 
 my $expected_error_code = 0;
diff -r f5754f5d165c -r 3aa1d93d1c10 tests/Test-idn-headers.px
--- a/tests/Test-idn-headers.px	Fri Sep 25 10:17:49 2009 -0700
+++ b/tests/Test-idn-headers.px	Sun Sep 27 11:29:01 2009 -0700
@@ -40,7 +40,7 @@
     },
 );
 
-my $cmdline = $WgetTest::WGETPATH . " --debug --iri -rH"
+my $cmdline = $WgetTest::WGETPATH . " --iri -rH"
     . " -e http_proxy=localhost:{{port}} http://start-here.com/start.html";
 
 my $expected_error_code = 0;
diff -r f5754f5d165c -r 3aa1d93d1c10 tests/Test-idn-meta.px
--- a/tests/Test-idn-meta.px	Fri Sep 25 10:17:49 2009 -0700
+++ b/tests/Test-idn-meta.px	Sun Sep 27 11:29:01 2009 -0700
@@ -41,7 +41,7 @@
     },
 );
 
-my $cmdline = $WgetTest::WGETPATH . " --debug --iri -rH"
+my $cmdline = $WgetTest::WGETPATH . " --iri -rH"
     . " -e http_proxy=localhost:{{port}} http://start-here.com/start.html";
 
 my $expected_error_code = 0;
diff -r f5754f5d165c -r 3aa1d93d1c10 tests/Test-idn-robots.px
--- a/tests/Test-idn-robots.px	Fri Sep 25 10:17:49 2009 -0700
+++ b/tests/Test-idn-robots.px	Sun Sep 27 11:29:01 2009 -0700
@@ -48,7 +48,7 @@
     },
 );
 
-my $cmdline = $WgetTest::WGETPATH . " --debug --iri -rH"
+my $cmdline = $WgetTest::WGETPATH . " --iri -rH"
     . " -e http_proxy=localhost:{{port}} --local-encoding=EUC-JP"
     . " http://$euc_jp_hostname/";
 
diff -r f5754f5d165c -r 3aa1d93d1c10 tests/Test-proxy-auth-basic.px
--- a/tests/Test-proxy-auth-basic.px	Fri Sep 25 10:17:49 2009 -0700
+++ b/tests/Test-proxy-auth-basic.px	Sun Sep 27 11:29:01 2009 -0700
@@ -25,7 +25,7 @@
     },
 );
 
-my $cmdline = $WgetTest::WGETPATH . " --debug --user=fiddle-dee-dee --password=Dodgson"
+my $cmdline = $WgetTest::WGETPATH . " --user=fiddle-dee-dee --password=Dodgson"
     . " -e http_proxy=localhost:{{port}} http://no.such.domain/needs-auth.txt";
 
 my $expected_error_code = 0;




diff -r ee7b9018a483 tests/Test-E-k-K.px
--- a/tests/Test-E-k-K.px	Fri Oct 09 02:23:26 2009 -0700
+++ b/tests/Test-E-k-K.px	Tue Oct 13 21:18:51 2009 -0400
@@ -61,7 +61,7 @@
     },
 );
 
-my $cmdline = $WgetTest::WGETPATH . " -d -r -nd -E -k -K http://localhost:{{port}}/index.php";
+my $cmdline = $WgetTest::WGETPATH . " -r -nd -E -k -K http://localhost:{{port}}/index.php";
 
 my $expected_error_code = 0;
 
diff -r ee7b9018a483 tests/Test-cookies-401.px
--- a/tests/Test-cookies-401.px	Fri Oct 09 02:23:26 2009 -0700
+++ b/tests/Test-cookies-401.px	Tue Oct 13 21:18:51 2009 -0400
@@ -29,7 +29,7 @@
     },
 );
 
-my $cmdline = $WgetTest::WGETPATH . " -d http://localhost:{{port}}/one.txt"
+my $cmdline = $WgetTest::WGETPATH . " http://localhost:{{port}}/one.txt"
     . " http://localhost:{{port}}/two.txt";
 
 my $expected_error_code = 6;
diff -r ee7b9018a483 tests/Test-ftp-bad-list.px
--- a/tests/Test-ftp-bad-list.px	Fri Oct 09 02:23:26 2009 -0700
+++ b/tests/Test-ftp-bad-list.px	Tue Oct 13 21:18:51 2009 -0400
@@ -29,7 +29,7 @@
     },
 );
 
-my $cmdline = $WgetTest::WGETPATH . " -d -nH -Nc -r ftp://localhost:{{port}}/";
+my $cmdline = $WgetTest::WGETPATH . " -nH -Nc -r ftp://localhost:{{port}}/";
 
 my $expected_error_code = 0;
 
diff -r ee7b9018a483 tests/Test-iri-list.px
--- a/tests/Test-iri-list.px	Fri Oct 09 02:23:26 2009 -0700
+++ b/tests/Test-iri-list.px	Tue Oct 13 21:18:51 2009 -0400
@@ -143,7 +143,7 @@
     },
 );
 
-my $cmdline = $WgetTest::WGETPATH . " --iri -d -i http://localhost:{{port}}/url_list.txt";
+my $cmdline = $WgetTest::WGETPATH . " --iri -i http://localhost:{{port}}/url_list.txt";
 
 my $expected_error_code = 0;
 
diff -r ee7b9018a483 tests/Test-iri.px
--- a/tests/Test-iri.px	Fri Oct 09 02:23:26 2009 -0700
+++ b/tests/Test-iri.px	Tue Oct 13 21:18:51 2009 -0400
@@ -186,7 +186,7 @@
     },
 );
 
-my $cmdline = $WgetTest::WGETPATH . " -d --iri --restrict-file-names=nocontrol -nH -r http://localhost:{{port}}/";
+my $cmdline = $WgetTest::WGETPATH . " --iri --restrict-file-names=nocontrol -nH -r http://localhost:{{port}}/";
 
 my $expected_error_code = 0;
 
